### PR TITLE
Doc processor 0.3.2

### PR DESCRIPTION
**Pull request by Jolanrensen**
Based on https://github.com/Jolanrensen/docProcessorGradlePlugin/releases/tag/v0.3.0, let's update the library too :)

### CODE REVIEW

File: core/build.gradle.kts [hunk 1]
```diff
@@ -199,15 +199,6 @@ val processKDocsMain by creatingProcessDocTask(
         .filterNot { pathOf("build", "generated") in it.path }, // Exclude generated sources
 ) {
     target = file(generatedSourcesFolderName)
-    processors = listOf(
```
Comment (Jolanrensen) [inline core/build.gradle.kts:202]: They are now the default setting

File: core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/annotations/ImportDataSchema.kt [hunk 2]
```diff
@@ -66,7 +66,7 @@ public annotation class JsonOptions(
      *     will be created.
      *
      * Example:
-     * `["""$["store"]["book"][*]["author"]"""]`
+     * `["""[*]["author"]"""]`
```
Comment (koperagen) [inline core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/annotations/ImportDataSchema.kt]: looks suspicious

File: core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/JsonPath.kt [hunk 3]
```diff
@@ -8,9 +8,9 @@ import java.io.Serializable
  * Supports just keys (in bracket notation), double quotes, arrays and wildcards.
  *
  * Examples:
- * `$["store"]["book"][*]["author"]`
+ * `[*]["author"]`
```
Comment (koperagen) [inline core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/JsonPath.kt:11]: same

File: core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/Submitting.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package org.jetbrains.kotlinx.dataframe.api;
+
+import kotlin.Unit;
+
+import java.io.File;
+
+public class Submitting {
+
+    public boolean sample() {
+        int number = 1;
```
Comment (koperagen) [inline core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/Submitting.java]: what is this?

File: core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/codeGen/GeneratedField.kt [hunk 5]
```diff
@@ -104,7 +104,7 @@ public class ValidFieldName private constructor(private val identifier: String,
                     .replace("]", "}")
                     .replace("`", "'")
                     .replace(";", " ")
-                    .replace("\\", " ")
+                    .replace("\", " ")
```
Comment (koperagen) [inline core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/codeGen/GeneratedField.kt]: a mistake

File: core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/JsonPath.kt (hunk 3 above)
Comment (koperagen) [reply core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/JsonPath.kt:11]: this one is actually not fixed

Comment (Jolanrensen) [reply core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/JsonPath.kt:11]: yup! thanks, that's the danger of introducing new character-based hooks... Good find!

File: core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/Submitting.java (hunk 4 above)
Comment (Jolanrensen) [reply core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/Submitting.java]: uncommitted temp file, will be removed :)

File: core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/codeGen/GeneratedField.kt (hunk 5 above)
Comment (Jolanrensen) [reply core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/codeGen/GeneratedField.kt]: already fixed locally :) thanks

File: core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/JsonPath.kt (hunk 3 above)
Comment (Jolanrensen) [reply core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/JsonPath.kt:11]: Actually, this needs a fix on the doc processor side. "`\$["store"]["book"][*]["author"]`" should solve it, but it seems the "`\`" isn't removed correctly.

**Pull request merged**